internal/resources/resource_host.go

```diff
@@ -597,7 +597,16 @@ func resourceMetalHostUpdate(d *schema.ResourceData, meta interface{}) (err erro
 	}
 
 	// set the network for default route
-	if host.NetworkForDefaultRoute, err = getNetworkDefRoute(d, p, &host); err != nil {
+	if nDefRoute := safeString(d.Get(hNetForDefaultRoute)); nDefRoute != "" {
+		if host.NetworkForDefaultRoute, err = getNetworkID(p, host.NetworkIDs, host.LocationID, nDefRoute); err != nil {
+			return err
+		}
+	}
+
+	// set the untagged network
+	if nUntagged := safeString(d.Get(hNetUntagged)); nUntagged == "" {
+		host.NetworkUntagged = nUntagged
```

Comment:
This is clearing untagged network selection?  If so --
```suggestion
		host.NetworkUntagged = ""
```